src/components/assets/styles/main.scss

```diff
@@ -0,0 +1,49 @@
+@import 'design';
```

Comment:
In the mobile view:

- [ ] There seems to be no hamburger menu, or some sort of menu to open the navigation.  Kindly add one.
- [ ] The mobile menu is getting overflowed by the cars on the main page:

![image](https://user-images.githubusercontent.com/57421823/213178480-63c890ea-2835-490a-96ae-5081739b7882.png)


Kindly correct this bug.  That way, your mobile app will be beautiful, and professional. 👍